v2/tools/asoctl/go.mod

```diff
@@ -2,17 +2,91 @@ module github.com/Azure/azure-service-operator/v2/tools/asoctl
 
 go 1.18
 
-// Needed to reference shared version numbering:
 replace github.com/Azure/azure-service-operator/v2 => ../../
```

Comment:
I think we can eventually do away with this in favor of a publicly exported `CreateScheme` method, but fine with this for now